analyze/analyses/paths/paths_entry.rs

```diff
@@ -0,0 +1,26 @@
+use std::cmp;
+
+#[derive(Debug, PartialEq, Eq)]
+pub(super) struct PathsEntry {
+    pub name: String,
+    pub size: u32,
+    pub children: Vec<PathsEntry>,
+}
+
+impl PathsEntry {
+    pub fn _count(&self) -> u32 {
+        1 + self.children.iter().map(|c| c._count()).sum::<u32>()
+    }
```

Comment:
This is included for the sake of the summary logic that can be enabled once this lands :)